DataConnectors/CyberArk/CyberArk Data Connector.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "CyberArk",
+    "title": "CyberArk Enterprise Password Vault (EPV) Syslog",
+    "publisher": "Cyber-Ark",
+    "descriptionMarkdown": "CyberArk Enterprise Password Vault generates an xml Syslog messages for every action taken against the Vault.  The EPV will send the xml messages through the Sentinel.xsl translator to be converted into CEF standard format and sent to a syslog staging server of your choice (syslog-ng, rsyslog). The Log Analytics agent installed on your syslog staging server will import the messages into Azure Log Analytics",
```

Comment:
1. Syslog message is right grammatically since it's 'generates an xml Syslog messages' which seems incorrect. Please fix.
2. It'd be great to also add a link to CyberArk product documentation.